hphp/runtime/vm/jit/vasm-arm.cpp

```diff
@@ -106,10 +106,8 @@ int64_t MSKTOP(int64_t value) {
 vixl::MemOperand M(Vptr p) {
   assertx(p.base.isValid());
   if (p.index.isValid()) {
-    assertx(p.disp == 0);
```

Comment:
Definitely need this assertion. If the Vptr was lowered and it still has a disp, then we need to know if we're about to throw it away.